### PR TITLE
Metalands Dapp

### PR DESCRIPTION
**Pull request by Murtagh300**
Please review, And Pull Request to show Metalands V Dapp in App Hub.

### CODE REVIEW

**Comment by Murtagh300** [comment]
@libotony I am not 100% sure i did it right. Please let me know if any changes are needed

**Comment by Murtagh300** [comment]
I also updated BlackVeMarket Logo & Description. Sorry was not meaning to do it in the Same PR as the MetaLands Request.

File: apps/com.metalandsv/manifest.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+    "name": "Metalands V",
+    "href": "https://MetaLandsV.art",
+    "desc": "By minting a MetaLands NFT, you become an integral piece of the BlackVeMarket puzzle, contributing to the future development of this thriving ecosystem. With each NFT minted, you support the evolution of BlackVeMarket and cement your place in its vibrant history.",
+    "category": "collectibles, utility",
```
Comment (XJWX89) [inline apps/com.metalandsv/manifest.json:5]: category must be **one** of the provided category.

**Review by XJWX89 — CHANGES_REQUESTED** [review]
hi @Murtagh300 , thank you for submitting the new application. However, we regret to inform you that it has not been accepted due to violation of [Rules # 7 -One application per submission](https://github.com/vechain/app-hub#rules) .Thank you for your understanding.

**Pull request closed without merging**